components/emoji_picker/emoji_picker.jsx

```diff
@@ -578,6 +578,7 @@ export default class EmojiPicker extends React.PureComponent {
                             aria-label={ariaLabel}
                             ref={this.emojiSearchInput}
                             className='emoji-picker__search'
+                            data-testid='emoji-input__search'
```

Comment:
camelCase testIds. can be `emojiInputSearch'.

Reply:
make sense, done